homeassistant/components/homekit/__init__.py

```diff
@@ -239,6 +240,10 @@ def start(self, *args):
         if not self.driver.state.paired:
             show_setup_message(self.hass, self.driver.state.pincode)
 
+        if len(self.bridge.accessories) > MAX_DEVICES:
+            _LOGGER.warning("More than %s devices added to homekit.",
+                            MAX_DEVICES)
```

Comment:
I agree that it shouldn't be that long. However, if I were to read this warning for the first time without background knowledge, I wouldn't know what to do. What do you think of:

`You have exceeded the device limit, which might cause issues. Consider using the filter option.`

Reply:
👍